---
linktitle: STAT35510 Statistical Algorithms for Single-Cell Omics and Related Techniques
weight: 2


# Page metadata.
title: STAT 35510 Statistical Algorithms for Single-Cell Omics and Related Techniques
date: "2025-01-06T00:00:00Z"
lastmod: "2018-09-09T00:00:00Z"
draft: false  # Is this a draft? true/false
toc: false  # Show table of contents? true/false
type: docs  # Do not modify.

# Add menu entry to sidebar.
# - name: Declare this menu item as a parent with ID `name`.
# - weight: Position of link in menu.
menu:
  stat35510_2025:
 #   name: Overview
    weight: 1

---



## Description

Single-cell sequencing is a revolutionary technique that allows the analysis of the genetic information within individual cells, providing unprecedented insights into cellular heterogeneity and diversity. This course aims to offer a comprehensive overview of the cutting-edge quantitative methods employed in analyzing single-cell sequencing data. Designed for graduate students with a statistical/quantitative background, the course requires no prior knowledge of biology or experience in analyzing genetics data. We will start with a gentle introduction to basic biological concepts relevant to understanding the data, coupled with a concise overview of single-cell sequencing technologies such as single-cell RNA-seq, single-cell ATAC-seq, spatial transcriptomics, CITE-seq, Perturb-seq, and more. Then, we will discuss common types of computational analyses, such as visualization, denoising, clustering, trajectory analyses, data integration, transfer learning, and the alignment of multi-omics data. A special emphasis will be placed on exploring deep learning models that have been designed for various tasks analyzing single-cell sequencing data. We will also address statistical considerations that arise, including appropriate distributional assumptions on the data, distribution-free tests, “post-estimation” inference, and causal inference.


## Course Materials
Week | Date | Topic | Slides 
---|---|---|---
1| 2025-01-07 | Introduction: basic biology, single-cell omics, scRNA-seq technique |  {{% staticref "materials/stat35510_2025/Lecture1.pdf" "newtab" %}}Lecture 1{{% /staticref %}} 
1| 2025-01-09 | scRNA-seq technique and count matrix QC |  {{% staticref "materials/stat35510_2025/Lecture2.pdf" "newtab" %}}Lecture 2{{% /staticref %}} 
2| 2025-01-14 | scRNA-seq noise and signal distributions |  {{% staticref "materials/stat35510_2025/Lecture3.pdf" "newtab" %}}Lecture 3{{% /staticref %}} 
2| 2025-01-16 | scRNA-seq analysis workflow, normalization, visualization |  {{% staticref "materials/stat35510_2025/Lecture4.pdf" "newtab" %}}Lecture 4{{% /staticref %}} 
3| 2025-01-21 | scRNA-seq clustering and cell type annotation |  {{% staticref "materials/stat35510_2025/Lecture5.pdf" "newtab" %}}Lecture 5{{% /staticref %}} 
3| 2025-01-23 | scRNA-seq denoising |  {{% staticref "materials/stat35510_2025/Lecture6.pdf" "newtab" %}}Lecture 6{{% /staticref %}} 
4| 2025-01-30 | scRNA-seq trajectory analysis |  {{% staticref "materials/stat35510_2025/Lecture7.pdf" "newtab" %}}Lecture 7{{% /staticref %}} 
5| 2025-02-04 | RNA velocity |  {{% staticref "materials/stat35510_2025/Lecture8.pdf" "newtab" %}}Lecture 8{{% /staticref %}} 
5| 2025-02-06 | Data integration and batch correction |  {{% staticref "materials/stat35510_2025/Lecture9.pdf" "newtab" %}}Lecture 9{{% /staticref %}}
6| 2025-02-13 | Reference mapping and transfer learning |  {{% staticref "materials/stat35510_2025/Lecture10.pdf" "newtab" %}}Lecture 10{{% /staticref %}}


